pkg/cache/basic/cache.go

```diff
@@ -50,6 +52,12 @@ func NewCache[T any](options ...Option) (*BasicCache[T], error) {
 		defaultTTL:       config.defaultTTL,
 	}
 
+	fmt.Println("Cache frequency is", config.cleanupFrequency)
+
```

Comment:
Probably a debug log message at most.

Reply:
we should probably add a linter to catch these :(